R/get_risktable.R

```diff
@@ -19,10 +19,10 @@ get_risktable <- function(x, ...){
 
 #' @param x an object of class `survfit` or `tidycuminc`
```

Comment:
> Could we reconcile the code for the survfit object and tidycuminc? I see a lot of duplication eg the way labels/statlist are reconciled. In this specific example I dont have any preference on how it is done, but It would be good to choose one piece of code. We prefer to keep the code very basic but efficient and don't rely too much on functions that can result in breaking changes in the future.

That is a great idea. I can work on that.

> The only main difference I might see is: the cleaning of the object - visr_tidy_tidycuminc => can this be a tidyme function?

The `visr_tidy_tidycuminc()` only does two things, 1. changes the confidence interval column names and the estimate column name, 2. add a strata column if there is no stratifying variable. Otherwise, it's the same functionality as `tidycmprsk::tidy()`. If you want to export the function, it'll be very easy to do! Just let me know!

```r
tidyme.tidycuminc <- function(x, ...) {
  tidycmprsk::tidy(x, ...) %>% visr_tidy_tidycuminc()
}
```